packages/react/src/components/DataTable/DataTable.types.ts

```diff
@@ -208,3 +228,21 @@ export interface DataTableProps<TData extends RowData> {
    */
   onSortingChange?: OnChangeFn<SortingState>;
 }
+
+export interface TotalsProps {
+  totalsData: TotalsDataObj;
+  totalsKeyword: string;
+}
+
+export type TotalsDataObj = Record<string, TotalsHeaderObj>;
+
+export interface TotalsHeaderObj {
+  value: number;
+  has_custom_html: boolean;
+  html: string;
+  links: Link[];
```

Comment:
what do these props do?

Reply:
These will provide user option to add links and html to total column

Reply:
I don't see it used anywhere

Reply:
@vitaliy-p44 I have updated these prop to use generic type